contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -359,9 +360,33 @@ mod tests {
         );
     }
 
+    const TEST_DATA_1: &str = "./testdata/airdrop_stage_1_test_data.json";
```

Comment:
I like it!

Reply:
Looking above, I don't see the command that generates these test data.
Can you please add this to the README of the ts side (or a comment here)?